package/Dockerfile

```diff
@@ -15,10 +15,15 @@ RUN mkdir -p /go/src/github.com/k3s-io/helm-set-status && \
     make -C /go/src/github.com/k3s-io/helm-set-status install
 
 FROM alpine:3.12
-RUN apk add -U --no-cache ca-certificates jq bash git
+WORKDIR /home/klipper-helm
+ENV HOME=/home/klipper-helm
 COPY --from=extract /usr/bin/helm_v2 /usr/bin/helm_v3 /usr/bin/tiller /usr/bin/
 COPY --from=plugins /root/.local/share/helm/plugins/ /root/.local/share/helm/plugins/
 COPY entry /usr/bin/
+RUN apk add -U --no-cache ca-certificates jq bash git && \
+    chown 1000:1000 -R /home/klipper-helm && \
+    chmod +x /usr/bin/entry
```

Comment:
not sure why this is necessary; the file in git is already executable and should keep the x bit when copied in?